servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Generator.java

```diff
@@ -1639,6 +1645,43 @@ private TypeSpec newServiceInterfaceSpec(final State state, final boolean blocki
                 .addStatement("return $L", blocking ? BLOCKING_METHOD_DESCRIPTORS : ASYNC_METHOD_DESCRIPTORS)
                 .build());
 
+        // generate default service methods
+        if (defaultServiceMethods) {
+            final List<MethodDescriptorProto> methodList = state.serviceProto.getMethodList();
+            for (int i = 0; i < methodList.size(); i++) {
+                final MethodDescriptorProto methodProto = methodList.get(i);
+                final ClassName inClass = messageTypesMap.get(methodProto.getInputType());
+                final ClassName outClass = messageTypesMap.get(methodProto.getOutputType());
+                final String methodName = sanitizeIdentifier(methodProto.getName(), true);
+                final String serviceName = isNullOrEmpty(state.serviceProto.getName()) ?
+                        "" : state.serviceProto.getName();
+                final String fullMethodName = serviceName + "/" + methodName;
+                final int methodIndex = i;
+                interfaceSpecBuilder.addMethod(newRpcMethodSpec(inClass, outClass, methodName,
+                        methodProto.getClientStreaming(),
+                        methodProto.getServerStreaming(),
+                        !blocking ? EnumSet.of(INTERFACE) : EnumSet.of(INTERFACE, BLOCKING),
+                        printJavaDocs, (__, c) -> {
```

Comment:
All methods at service level (`TesterService` and `BlockingTesterService`) should have `@Override` annotation